samples/csharp_dotnetcore/50.teams-messaging-extensions-search/Bots/TeamsMessagingExtensionsSearchBot.cs

```diff
@@ -99,5 +134,131 @@ private async Task<IEnumerable<(string, string, string, string, string)>> FindPa
+                {
+                    Type = "result",
+                    AttachmentLayout = "list",
+                    Attachments = new List<MessagingExtensionAttachment> { attachment }
+                }
+            };
+        }
+
+        public MessagingExtensionResponse GetConnectorCard()
+        {
+            string filepath = "./connectorCard.json";
+
+            var previewcard = new ThumbnailCard
+            {
+                Title = "O365 Connector Card",
+                Text = "Please select to get Connector card"
+            };
+
+            var connector = FetchConnector(filepath);
+            var attachment = new MessagingExtensionAttachment
+            {
+                ContentType = O365ConnectorCard.ContentType,
+                Content = connector.Content,
+                Preview = previewcard.ToAttachment()
+            };
+
+            return new MessagingExtensionResponse
+            {
+                ComposeExtension = new MessagingExtensionResult
+                {
+                    Type = "result",
+                    AttachmentLayout = "list",
+                    Attachments = new List<MessagingExtensionAttachment> { attachment }
+                }
+            };
+        }
+
+        public static Attachment FetchAdaptive(string filepath)
+        {
+            var adaptiveCardJson = File.ReadAllText(filepath);
+            var adaptiveCardAttachment = new Attachment
+            {
+                ContentType = AdaptiveCard.ContentType,
+                Content = JsonConvert.DeserializeObject(adaptiveCardJson),
+            };
+
+            return adaptiveCardAttachment;
+
+        }
+
+        public Attachment FetchConnector(string filepath)
+        {
+            var connectorCardJson = File.ReadAllText(filepath);
+            var connectorCardAttachment = new MessagingExtensionAttachment
+            {
+                ContentType = O365ConnectorCard.ContentType,
+                Content = JsonConvert.DeserializeObject(connectorCardJson),
+
+            };
+
+            return connectorCardAttachment;
+        }
+
+        public MessagingExtensionResponse GetResultGrid()
+        {           
+            var files = Directory.GetFiles("wwwroot");
+            List<string> imageFiles = new List<string>();
+
+            foreach (string filename in files)
+            {
+                if (Regex.IsMatch(filename, @".jpg"))
+                    imageFiles.Add(filename);
+                
+            }
+
+            List<MessagingExtensionAttachment> attachments = new List<MessagingExtensionAttachment>();
+
+            foreach (string img in imageFiles)
+            {
+                var image = img.Split("\\");
+                string url = _configuration["BaseUrl"];
+                var thumbnailCard = new ThumbnailCard();
+                thumbnailCard.Images = new List<CardImage>() { new CardImage(_configuration+ image[1]) };
```

Comment:
This does not look correct:
```new CardImage(_configuration+ image[1])```

Please deploy, and try this sample.

Reply:
Yes fixed this there was issue with _configuration which is corrected now, I'm reading the images from folder and splitting the image name